client/components/activity-card-list/index.jsx

```diff
@@ -321,6 +322,7 @@ class ActivityCardList extends Component {
 				<QueryRewindPolicies siteId={ siteId } />
 				<QueryRewindCapabilities siteId={ siteId } />
 				<QueryRewindState siteId={ siteId } />
+				<QueryJetpackCredentialsStatus siteId={ siteId } role="main" />
```

Comment:
Should this be rendered in the component where we actually use the selectors to render the credentials?

Reply:
> Should this be rendered in the component where we actually use the selectors to render the credentials?

The components that use the `areJetpackCredentialsInvalid()` selector are action buttons, and we use them to decide whether we need to disable buttons. Given there may be multiple action buttons in the same page, we considered a better option to render this `QueryJetpackCredentialsStatus` in a parent component once.